src/pages/LoginForm.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from "react";
+import {
+  Flex,
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  CircularProgress,
+  Text,
+  Link,
+} from "@chakra-ui/core";
+
+import ErrorMessage from "../components/ErrorMessage";
+
+const LoginForm = () => {
+  const [state, setState] = useState({
+    email: "",
+    password: "",
+    isLoading: false,
+    error: "",
+    signUp: true,
+  });
+  const { email, confirmPassword, password, isLoading, error, signUp } = state;
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    alert(`Email: ${email} & Password: ${password}`);
```

Comment:
We will have to remove this too, but I think we can do this after api endpoints are built.

Reply:
Yes, that's the idea